crates/cxx-qt-lib-headers/include/core/qtlogging.h

```diff
@@ -0,0 +1,54 @@
+// clang-format off
```

Comment:
(also FYI best to fork the repository and manage your branches there rather than adding them to the kdab repository)